lib/ansible/modules/windows/win_firewall_rule.py

```diff
@@ -100,6 +100,11 @@
       - Defaults to C(domain,private,public) when creating a new rule.
     type: list
     aliases: [ profile ]
+  icmptypecode:
+    description:
+      - The ICMP Types and codes for the rule.
```

Comment:
I see you have a special check for `any` in the actual code, what's the significance of that?